### DISCUSSION TITLE
Manager + Forward/Heavy node combined distributed deployment?

### DISCUSSION THREAD

**Discussion by csanate** [Security-Onion-Solutions/securityonion#6100] [Unsupported Versions] [answered]
Hello! Is there a way to create a distributed deployment with a manager node that is also a forward or heavy node? I was under the impression that this would be the same as a heavy node, but as I am running through the setup, this does not seem to be the case.

During setup for the first box, I am selecting:

- Install Type: Distributed
- Distributed Node Type: HeavyNode
- Manager Server Hostname: The hostname of this box that I am provisioning

Once it gets to where it tries to connect to the manager node with the `soremote` user, it connects to 127.0.0.1, but will not accept the password. If I check /etc/passwd on the box, I see there is no `soremote` user. After reviewing things, I believe this is because the manager node role is what creates this user. 

Is there any way to make a heavy or forward node a manager as well? I understand the implications of heavy nodes, this is a small deployment that sees less than 1.5Mbps of traffic, and I have ample processing power. I have an additional forward node that I would like to point at this initial manager/heavy node. This also sees a small amount of throughput. What I don't have is a 3rd box that can run as a standalone manager. 

Could I spin this up as a manager node and then somehow apply the additional heavy or forward roles to it?

Thanks!

**Comment by dougburks** [comment, marked as answer]
> * Distributed Node Type: HeavyNode

When you select HeavyNode, you are telling Setup that you already have a separate box configured as your Manager.

> Is there any way to make a heavy or forward node a manager as well?

Seems like the [Standalone](https://docs.securityonion.net/en/2.3/architecture.html#standalone) option would work for your needs.  This is Manager + Search + Sensor all on one box and allows you join separate forward nodes as well.

**Comment by csanate** [comment]
Thank you, sir!
